### DISCUSSION TITLE
Is it possible to piggyback off of a wired controller?

### DISCUSSION THREAD

**Discussion by MarkinBrisbane** [revk/xESP32-Faikout#570] [Q&A]
Hi all - I am wondering if it is possible to piggyback a faikin off of a wired controller?

My BRP15A61 controller died (it's still on the wifi but is is stuck with a failed firmware update from Daikin or was bricked by them to retire the product).  It is almost impossible to climb into the roof cavity to get to the current controller and connection, so that leaves me wondering if I can somehow piggyback off the wall unit?

My wall unit is a Daikin BRC1E61 - https://www.manualslib.com/manual/855469/Daikin-Brc1e61.html#manual and I know I can get to the wiring behind it.

So my question is... is it possible to somehow piggyback of the wired controller? 

If this has already been answered, then a humble apology is due, I did have a good look and search but may have used the wrong terms

Also I did some further digging and found another device which seems to suggest that piggybacking off a wired controller is a valid configuration: https://hmsnetworks.blob.core.windows.net/nlw/docs/default-source/products/intesis/manuals-and-guides---installation-guides/installation-sheet-inwmpdai001r000.pdf?sfvrsn=804d50d7_13

**Comment by revk** [comment]
They are a different protocol, sorry.
